insights/tests/client/connection/test_test_connection.py

```diff
@@ -0,0 +1,76 @@
+from logging import ERROR
+from unittest.mock import patch
+
+from pytest import fixture
+from pytest import mark
+from requests.exceptions import ConnectTimeout
+from requests.exceptions import ReadTimeout
+from requests.exceptions import SSLError
+
+from insights.client.config import InsightsConfig
+from insights.client.connection import InsightsConnection
+
+EXCEPTION_MESSAGE = "exception"
```

Comment:
This feels unnecessary.

Non-blocking comment.

Reply:
You were right: the constant has only been used only to build the exceptions to be thrown.

However, I intended to use it to check the actual output in the _test_request_failed_print_exception_, but I put the actual exception, not its message into the string interpolation. I replaced the exception instance with the message there. I am aware that it’s testing how Python internally converts exceptions into strings, yet the test verifies the actual output string – and there the plain message is expected, not _any_ representation of the exception object, although they happen to be the same and that is leveraged in the actual code that build the output.